outputs.tf

```diff
@@ -123,3 +123,19 @@ output "autoscaler_account" {
 output "cluster_api_mode" {
   value = var.cluster_api_mode
 }
+
+output "ebs_csi_account" {
+  value = module.ebs.ebs_csi_account
+}
+
+output "k8s_version" {
```

Comment:
Is this something that Azure, GCP, and OSS should do/a ticket be created for consistency?

Though it's used in DaC only for AWS-EBS at the moment, If the variable `k8s_version` is not outputted for those providers we can't use `K8S_VERSION` in DaC
This line:
https://github.com/sassoftware/viya4-deployment/pull/298/files#diff-2fa74c16b93705662680b1545375b17e4a573a904b86bda46b94fe5e48447438R105

Reply:
The k8s_version, yes. All other items are cloud specific.